### ISSUE TITLE
Mouse does not work in full screen mode

### ISSUE THREAD

**Issue by kevinjardine** [langurmonkey/gaiasky#210]
Under Java 9 and Windows 10, the mouse does not work in full screen mode for me. It moves back and forth (x direction) but there is no up and down (y direction) movement.

The mouse works fine in a window, even a maximized one.

This is on a 4K monitor in case it matters.

**Comment by langurmonkey** [comment]
Gaia Sky has never been tested with Java 9 yet, so problems are to be expected. I will sort this out whenever I have some time to look into it.

**Comment by langurmonkey** [comment]
Is this bug still present in version 2.0.0?

**Comment by kevinjardine** [comment]
Yes, I tested it after installed from github. So I run in windowed mode. No
real limitation for me but might be for others.

**Comment by langurmonkey** [comment]
Just tested this (Windows 10, 4K screen, Java 10) and it worked just fine. Are you still using Java 9?

**Comment by kevinjardine** [comment]
Yes, Java 9. I will upgrade soon.
